src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -502,7 +504,7 @@ function MoneyRequestView({
                     <MenuItemWithTopDescription
                         description={translate('common.description')}
                         shouldParseTitle
-                        title={TransactionUtils.getDescription(updatedTransaction ?? null) ?? transactionDescription}
+                        title={updatedTransactionDescription === '' ? transactionDescription : updatedTransactionDescription}
```

Comment:
Using this approach to avoid addint lint exception that is something we need to avoid doing for react-compiler to work